src/components/SecondaryPanes/index.js

```diff
@@ -213,6 +213,10 @@ class SecondaryPanes extends Component<Props, State> {
   }
 
   getWatchItem(): AccordionPaneItem {
+    if (this.state.showExpressionsInput) {
+      prefs.expressionsVisible = true;
+    }
```

Comment:
what does this do?

Reply:
Bailed on that change, found a cleaner way.